cfme/utils/events.py

```diff
@@ -257,76 +107,60 @@ def matches(self, evt):
             return True
 
     def add_attrs(self, *attrs):
-        """
-        event consists of attributes like event_type, etc.
-        this method allows to add an attribute to event
-        """
-        if isinstance(attrs, Iterable):
-            for attr in attrs:
-                if attr.name == 'target_name':
-                    # this is artificial attr which will be converted to target_id during matching
-                    self.event_attrs[attr.name] = attr
-                elif attr.name in self._default_attrs:
-                    # type check was removed because sqlalchemy's python_type
-                    # and type of returned values are different
-                    self.event_attrs[attr.name] = attr
-                else:
-                    logger.warning('The attribute {} type {} is absent in DB '
-                                   'or type mismatch.'.format(attr.name, attr.type))
-        else:
+        """ Adds an EventAttr to event."""
+        if not isinstance(attrs, Iterable):
```

Comment:
this check doesn't make sense, since attrs is a catchall for any positional argument (it also makes no sense in the original code

Reply:
@RonnyPfannschmidt OK, I have doubts whether to remove it or not.